src/components/Secrets/utils/service-account-utils.ts

```diff
@@ -101,3 +116,105 @@ export const unLinkSecretFromServiceAccount = async (secret: SecretKind, namespa
     ],
   });
 };
+
+export const getLinkedServiceAccount = async (secret: SecretKind) => {
```

Comment:
nit: since we're returning here an array of "service accounts", maybe we could make it plural :) `getLinkedServiceAccounts`